packages/connectivity/src/scp-cf/destination/http-proxy-util.ts

```diff
@@ -264,21 +268,34 @@ export function proxyAgent(
     ...options
   };
 
+  const { protocol, host, port, ...agentOptions } = agentConfig;
+  const urlConfig = getProxyUrl({ protocol, host, port });
```

Comment:
The function is called `getProxyUrl` perhaps I would call the const variable `proxyUrl`